sources/packages/backend/apps/db-migrations/src/migrations/1680120153698-DesignationAgreementsTableAlterSubmittedData.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class DesignationAgreementsTableAlterSubmittedData1680120153698
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData(
+        "Update-col-submitted_data-drop-not-null.sql",
```

Comment:
Minor thing. We never use `_` for sql file names. (I saw there is only one file which is exceptional).

Reply:
I will fix it.